src/main/java/org/apache/commons/io/input/ReaderInputStream.java

```diff
@@ -129,6 +128,9 @@ public ReaderInputStream(final Reader reader) {
      * Constructs a new {@link ReaderInputStream} with a default input buffer size of {@value #DEFAULT_BUFFER_SIZE}
      * characters.
      *
+     * <p>The encoder created for the specified charset will use {@link CodingErrorAction#REPLACE} for malformed input
+     * and unmappable characters.
+     *
```

Comment:
Close HTML tags.

Reply:
Have also moved the `<p>` on a separate line to match the style used above for the class documentation.